packages/dropdowns/src/elements/Dropdown/Dropdown.spec.tsx

```diff
@@ -241,7 +241,7 @@ describe('Dropdown', () => {
       container.querySelector('input')!.readOnly = false;
       userEvent.type(container.querySelector('input')!, 't');
 
-      expect(onInputValueChangeSpy.mock.calls[1][0]).toBe('t');
+      expect(onInputValueChangeSpy.mock.calls[0][0]).toBe('t');
```

Comment:
Since we no longer clear the input, this test makes a single mock fn call now. Previously, it was making 2 calls before, first with "" and second with the typed character, which was less than ideal anyway.